app/tasks/raster_tile_cache_assets/raster_tile_cache_assets.py

```diff
@@ -65,7 +65,7 @@ async def raster_tile_cache_asset(
                 AssetType.raster_tile_set,
                 source_asset.creation_options,
             )
-        ).replace("/geotiff", "/gdal-geotiff")
```

Comment:
This `replace` looks like it was tacked on, originally. Now that it's being removed, what assets _could_ be affected?

Reply:
Yeah, I had originally slipped it in as an aspirational optimization (i.e. I have no idea if it actually made a difference). I am trying to revert it now because I feel self-conscious about tacking it on, and I haven't entirely thought through whether or not it will always work (I think it does by luck). The change should not affect any existing assets, though, only new ones which are created.

Reply:
Just so I understand, when creating a tile cache, it tries to use the gdal-geotiff assets, but you're changing it to just use the normal geotiff assets because the WM assets don't have gdal-geotiff assets?

Reply:
> Just so I understand, when creating a tile cache, it tries to use the gdal-geotiff assets, but you're changing it to just use the normal geotiff assets because the WM assets don't have gdal-geotiff assets?
Yeah, we don't create the gdal-geotiff assets with the resample script, and if there's any benefit to doing so it's minimal. And this only changed to using gdal-geotiffs in one isolated instance. I think better to be consistent unless we adopt using gdal-geotiffs comprehensively.